src/popup/qrcode.html

```diff
@@ -22,6 +22,9 @@
 				<img class="icon-dismiss invisible" src="/common/img/close.svg" width="24" height="24" tabindex="0" data-i18n data-i18n-aria-label="__MSG_dismissIconDescription__"></span>
 			</div>
 		</div>
+		<div>
+			<p id="zoomTooltip">Alt +/- to zoom</p>
```

Comment:
That would need to be localized, otherwise, it's static in all languages.

But let's first see whether that is a the appropriate approach.

For more information, please [see the MDN guide on how to localize WebExtensions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales) and [our internationalization (i18n) guide](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#internationalisation-i18n).